helpful.el

```diff
@@ -1333,10 +1332,11 @@ LIBRARY-NAME takes the form \"foo.el\" , \"foo.el\" or
 \"src/foo.c\".
 
 If .elc files exist without the corresponding .el, return nil."
-  (when (member (f-ext library-name) '("c" "rs"))
+  (when (member (file-name-extension library-name) '("c" "rs"))
     (setq library-name
-          (f-expand library-name
-                    (f-parent find-function-C-source-directory))))
+          (expand-file-name library-name
```

Comment:
This is slightly different in that it uses file-name handlers, but I'm pretty sure that's what we want (e.g., if the source directory is on a remote machine, in a docker container, etc.).